### DISCUSSION TITLE
Is there a database of security advisories about vulnerabilities in packages installed through Brew ?

### DISCUSSION THREAD

**Discussion by Damian-Mangold** [Homebrew/discussions#5310] [Everyday usage]
Dear community,

I am writing to inquire whether there are any Security Advisories Database regarding vulnerabilities in packages installed via Brew. I am seeking a solution similar to those provided by other package managers, such as NPM or pip, where vulnerabilities are tracked through [OSV](https://osv.dev/) for example.

I appreciate your attention and assistance in this matter.

**Comment by SMillerDev** [comment]
Maybe repology.org has one, but in general Homebrew behaves nothing like npm or pip so it generally also does not have features they have
